library/src/main/java/in/krharsh17/barview/Bar.java

```diff
@@ -13,10 +13,20 @@
 
 class Bar extends View {
 
+    /**
+     * It can be called by a anyone directly from code to create a new instance of the view.
+     * This constructor doesn’t have access to XML attributes, so you have to fill the parameters manually, using setters.
+     * @param context of the activity
+     */
```

Comment:
```suggestion
    /**
     * It can be called by anyone directly from code to create a new instance of the view.
     * This constructor doesn’t have access to XML attributes, so you have to fill the parameters manually, using setters.
     * @param context of the activity
     */
```